Bryn Mawr Classical Review.js

```diff
@@ -59,6 +76,8 @@ function doWeb(doc, url) {
 	Zotero.Utilities.processDocuments(arts, function(doc) {
 		var item = new Zotero.Item("journalArticle");
 		var title = doc.evaluate('//h3/i', doc, nsResolver, XPathResult.ANY_TYPE, null).iterateNext().textContent;
+		item.publicationTitle = "Bryn Mawr Classical Review";
+		item.journalAbbreviation = "BMCR";
```

Comment:
OK, did this one and updated the tests too.

I'll try to get to the other ones, though I'm not sure I can handle them all.